stories/Components/Buttons/ShareButtons/share-buttons.scss

```diff
@@ -2,109 +2,113 @@
 // @import "../../../assets/scss/_variables";
 
 // main wrapper
-.mg-share-buttons-wrapper {
-  max-width: 250px;
-  background-color: rgba($mg-color-button-background, 0.2);
-  display: flex;
-  flex-direction: column;
-
-  // label SHARE THIS
-  .mainLabel {
-    padding: $mg-spacing-100;
-    padding-bottom: 0;
-    font-family: $mg-font-family;
-    color: $mg-color-text;
-    font-weight: 600;
-    font-size: $mg-font-size-300;
+section {
```

Comment:
I don't think we need the `<section>` element here, and a few other minor things:

- we could generally flatten the Sass nesting (.e.g shouldn't matter if `.mg-share-buttons` is a child of `.mg-share-wrapper` in the sass and creates specificity that isn't needed)
- would be good to limit the use of html elements in the Sass -- primarily as other systems may be less flexible and, for example, not be able to use a `header` element.

Reminds me that we need to write these stylistic things down somewhere ... largely a BEM methodology, but we also still need to catch up on tailwind.

Reply:
I might not be following fully, what do we gain by enforcing a specificity of `section.mg-share-wrapper`?

Happy to do a quick call, as i know a lot can get lost in text. Just ping me on teams if so.